tests/subsys/zigbee/osif/nvram/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Use software cryptography on nRF5340
+CONFIG_CRYPTO=n
+CONFIG_CRYPTO_NRF_ECB=n
+CONFIG_TINYCRYPT=y
+CONFIG_CTR_DRBG_CSPRNG_GENERATOR=y
+CONFIG_ZIGBEE_USE_SOFTWARE_AES=y
```

Comment:
I'm not 100% sure, but to my understanding, only the unit tests from the `zboss_api` directory have the `CONFIG_ZIGBEE` enabled. Why there is no issue in enabling this Kconfig, but there was in `crypto` tests?

Reply:
`prj.conf ` enables `CONFIG_ZIGBEE` - this `.conf` file is treated like an overlay config file, `prj.conf` is common for all board

Reply:
This is basically how we manage .conf files for Zigbee Shell sample